kart/meta.py

```diff
@@ -215,7 +215,7 @@ def _parse(key, value):
     )
 
 
-@click.command("commit-files", hidden=True, cls=KartCommand)
+@click.command("commit-files", cls=KartCommand)
```

Comment:
I'd leave it hidden because I don't think we have a real use case for it

Instead we should add an `--amend` option to `commit`; i'm not really sure why we don't have that yet